assets/dev/js/frontend/handlers/image-carousel.js

```diff
@@ -106,6 +108,9 @@ export default class ImageCarousel extends elementorModules.frontend.handlers.Sw
 
 		const Swiper = elementorFrontend.utils.swiper;
 
+		// Fix: When adding margin to column of media-carousel widget it breaks the UI [ED-4915]
+		this.elements.$widgetWrapper.width( 'calc( 100% - ( var( --e-column-margin-left, 0px ) + var( --e-column-margin-right, 0px ) ) )' );
+
```

Comment:
@guyc-E - The problem is not just with the 'Image Carousel', it also happens in the 'Media Carousel'.
Please consider using "overflow: hidden/auto' on the same div that the column margins apply to.

Reply:
I created a new task for "Media Carousel" and "Testimonial Carousel" for Pro.